src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1839,6 +1839,77 @@ public void canCreateMultipleLoansWithServicePoints()
     });
 
   }
+  
+  @Test
+  public void canGetPagedLoansWithItemsExceedingDefaultLimit()
```

Comment:
Given that the default page limit is 10, why is there a need fo create hundreds of loans to demonstrate this fix?

Is it to demonstrate exceeding the size of a query string parameter? If so, I think this could be renamed to reflect that.

If the goal is to be able to fetch 200 loans at once, then why bother to fetch smaller batches before that?

Reply:
Isn’t the default up to the storage module?